webrtc/RTCPeerConnection-setRemoteDescription.html

```diff
@@ -47,6 +47,43 @@
       assert_unreached('Error ' + e.name + ': ' + e.message);
     }));
   }, 'Triggers ontrack when called with a remote description and the MSID is parsed.');
+
+  promise_test(() => {
+    const pc = new RTCPeerConnection();
+    return pc.setRemoteDescription(
+      {'type': 'offer', 'sdp': 'bogus'})
+    .then(() => {
+      assert_unreached('Bogus SDP should not be accepted');
+    })
+    .catch(e => {
+      assert_equals(e.name, 'InvalidAccessError');
```

Comment:
Yeah that's not clear. I'll raise an issue. Also realized the test should throw RTCError instead of InvalidAccessError due to malformed syntax.